index.d.ts

```diff
@@ -157,9 +159,14 @@ export interface ModifierInvocation extends BaseASTNode {
 }
 export interface FunctionDefinition extends BaseASTNode {
   type: 'FunctionDefinition';
-  name: string;
+  name?: string;
```

Comment:
@Leeleo3x Can you give an example when `name` is undefined?

Reply:
```
constructor () {}
```
or
```
function () payable {} 
```

Reply:
@Leeleo3x For fallback (default) function, it has value of `""` (empty string). I will try to see constructor later.

Reply:
You may check the code

https://github.com/federicobond/solidity-parser-antlr/blob/master/src/ASTBuilder.js#L168